pkg/kubelet/rkt/rkt.go

```diff
@@ -51,8 +51,8 @@ import (
 )
 
 const (
-	acVersion             = "0.6.1"
-	rktMinimumVersion     = "0.8.1"
+	acVersion             = "0.7.1"
+	supportedRktVersion   = "0.9.0"
```

Comment:
> @yujuhong rkt should guarantee 0.9.1 won't break 0.9.0, but not 0.10.0 afaics. If 0.10.0 will fail in an unexpected way, I think we should just abort here?

We've had relevant discussions several times, and we have people supporting different approaches. I will try to summarize what I *think* was the direction that we're going (@dchen1107 @bgrant0607 in case my interpretation is not accurate):

For every k8s/kubelet release, we want the following rkt/docker versions:
 - A minimum supported version: any version smaller than this is not supported, and kubelet should fail.
 - A recommended version: the version we test, verify, and potentially ship with the new kubelet in the images. 

If a user wants to use an above minimum requirement but not the recommended version for various reasons (bug fixes, or just to experiment), kubelet should warn clearly but not fail completely (i.e., use it at your own risk).

I know rkt is not yet v1.0, and is going through a lot of changes, but supporting exactly one version is too strict.
+1 for relaxing the constraint to at least support all 0.9.x versions.